src/test/java/uk/gov/hmcts/darts/retention/service/impl/RetentionGetPolicyTypesServiceImplTest.java

```diff
@@ -77,6 +83,39 @@ void shouldSuccessfullyReturnListOfRetentionPolicyTypes() {
 
     }
 
+    @Test
+    void shouldSuccessfullyReturnPolicyType() {
+        setupStubs();
+
+        when(retentionPolicyTypeRepository.findById(anyInt())).thenReturn(Optional.of(getRetentionPolicyTypeEntity()));
```

Comment:
Is better to set up the stub with specific values where possible and also where there isn't a corresponding verify?

Reply:
will address.